### PR TITLE
release 3.5.3

### PR DESCRIPTION
**Pull request by kobenguyent**
## Motivation/Description of the PR
Release 3.5.3

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
+## 3.5.3
+
+🛩️ Features
+
+* [Playwright]: providing more steps to check the network traffics #3748 - by @ngraf @KobeNguyenT
```
Comment (pablopaul) [inline CHANGELOG.md]:
```suggestion
* [Playwright] Provide new steps to check network traffic #3748 - by @ngraf @KobeNguyenT
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+    parameters: {
+      width: '1919',
+      height: '1138',
+    },
+  });
+```
+
+🐛 Bugfix
+
+* [retryStepPlugin] fix the retry step doesn't work when using global retry #3768 - by @KobeNguyenT
```
Comment (pablopaul) [inline CHANGELOG.md]:
```suggestion
* [retryStepPlugin] Fix retry step when using global retry #3768 - by @KobeNguyenT
```

File: package.json [hunk 3]
```diff
@@ -153,4 +153,4 @@
     "npm": ">=5.6.0"
   },
   "es6": true
-}
```
Comment (pablopaul) [inline package.json]: @kobenguyent unneeded change, pls remove

**Review by DavertMik — APPROVED** [review]

**Comment by DavertMik** [comment]
Thank you @kobenguyent 
Changelog looks wonderful!

**Pull request merged**